sass/directives/02_base_components/button/_button.scss

```diff
@@ -193,6 +193,27 @@ $massive-button-horizontal-padding: $base-input-font-size * 2;
   }
 }
 
+@mixin signup-button {
```

Comment:
I feel that this naming may lock us in to its use case. We have examples of this happening across Employer that we still haven't been able to unravel, and I feel button styles are very frequently re-used. Should we go with the prior convention of just giving buttons numbered names?

Reply:
@toastercup I'm fine with changing this. I'd go with `senary-button` for this. It's not being used anywhere else in the site, so placing it at 6 would be fine. But the buttons are tiered in a way...

Reply:
:+1: